Essentials/src/com/earth2me/essentials/Settings.java

```diff
@@ -1540,4 +1550,18 @@ private boolean _allowOldIdSigns() {
     public boolean allowOldIdSigns() {
         return allowOldIdSigns;
     }
+
+    private boolean isWaterSafe;
+
+    private boolean _isWaterSafe() {
+        boolean isWaterSafe = config.getBoolean("is-water-safe", false);
```

Comment:
I'd rename this to `_isWaterSafe` to aid readability.

Reply:
What are you proposing to rename? The method or the field or the config key?

Reply:
The variable inside the method. It has the same name as the field above but it isn't clear at first glance.